src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -15,8 +15,8 @@ public class CreditCard extends BaseCreditCard {
      * @return CreditCardFund object.
      * @throws EasyPostException when the request fails.
      */
-    public static CreditCardFund fund(String amount, CreditCardPriority primaryOrSecondary) throws EasyPostException {
-        return fund(amount, primaryOrSecondary, null);
+    public static boolean fund(String amount, CreditCardPriority primaryOrSecondary) throws EasyPostException {
+        return true;
```

Comment:
This is incorrect, you're returning `true` instead of calling the function. it should be something like this `return fund(amount, primaryOrSecondary, null)`;